services/shhext/service_test.go

```diff
@@ -34,6 +38,14 @@ func (t handlerMock) EnvelopeExpired(hash common.Hash) {
 	t.expirations <- hash
 }
 
+func (t handlerMock) MailServerRequestCompleted(hash common.Hash) {
+	t.requestsCompleted <- hash
+}
+
+func (t handlerMock) MailServerRequestExpired(hash common.Hash) {
+	t.requestsExpired <- hash
```

Comment:
It is never checked which indicates that the expiration flow is not tested.

Reply:
I tested that part in the tracker but I'll add more test in the service as well!